packages/web-app-files/src/components/SideBar/SideBar.vue

```diff
@@ -214,9 +209,9 @@ export default {
         return
       }
 
-      if (newFile.id !== oldFile?.id) {
-        this.fetchFileInfo()
-      }
+      // always update even if the id is unchanged
```

Comment:
not sure if the comment is needed, maybe if someone reads it in the future it could be hard to understand without the context from the past.

Reply:
Yeah, true, but noone should pass this and think "Oh I can optimize this" ...  on the other hand we have a regression test for that now, so we can probably indeed remove it.

Reply:
Removed